server/schemas/field.graphql

```diff
@@ -27,6 +27,7 @@ type SchemaField {
   multiple: Boolean!
   unique: Boolean!
   required: Boolean!
+  isTitle: Boolean
```

Comment:
```suggestion
  isTitle: Boolean!
```

I am concerned that only this field has Is in the name, but it would be confusing to name it title, so this is fine.